docs/sources/datasources/prometheus/_index.md

```diff
@@ -143,39 +111,29 @@ To connect the Prometheus data source to Amazon Managed Service for Prometheus u
 
 If you run Grafana in an Amazon EKS cluster, follow the AWS guide to [Query using Grafana running in an Amazon EKS cluster](https://docs.aws.amazon.com/prometheus/latest/userguide/AMP-onboard-query-grafana-7.3.html).
 
-### Configure exemplars
+## Exemplars
+
+Exemplars are unique to Prometheus. Exemplars associate higher-cardinality metadata from a specific event with traditional time series data. See [Introduction to exemplars](https://grafana.com/docs/grafana/latest/fundamentals/exemplars/) in Prometheus documentation for detailed information on how they work.
 
 {{% admonition type="note" %}}
 Available in Prometheus v2.26 and higher with Grafana v7.4 and higher.
 {{% /admonition %}}
 
 Grafana 7.4 and higher can show exemplars data alongside a metric both in Explore and in Dashboards.
-Exemplars associate higher-cardinality metadata from a specific event with traditional time series data.
 
 {{< figure src="/static/img/docs/v74/exemplars.png" class="docs-image--no-shadow" caption="Screenshot showing the detail window of an Exemplar" >}}
 
-Configure Exemplars in the data source settings by adding external or internal links.
-
-{{< figure src="/static/img/docs/v74/exemplars-setting.png" class="docs-image--no-shadow" caption="Screenshot of the Exemplars configuration" >}}
-
-## Query the data source
+To configure exemplars see [Exemplars](/docs/grafana/latest/datasources/prometheus/configure-prometheus-data-source/#exemplars).
 
-You can create queries with the Prometheus data source's query editor.
+<!-- Configure Exemplars in the data source settings by adding external or internal links. -->
 
-For details, refer to the [query editor documentation]({{< relref "./query-editor/" >}}).
-
-## Use template variables
-
-Instead of hard-coding details such as server, application, and sensor names in metric queries, you can use variables.
-Grafana lists these variables in dropdown select boxes at the top of the dashboard to help you change the data displayed in your dashboard.
-Grafana refers to such variables as template variables.
-
-For details, see the [template variables documentation]({{< relref "./template-variables/" >}}).
+{{< figure src="/static/img/docs/prometheus/exemplars-10-1.png" max-width="500px" class="docs-image--no-shadow" caption="Exemplars" >}}
```

Comment:
What is this image showing? Should there be an intro to the image, like "The following image shows..."?